wasm/solana-client/Cargo.toml

```diff
@@ -30,7 +30,12 @@ solana-sdk = "~1.18"
 thiserror = "1"
 wasm-bindgen = { version = "0.2.88", optional = true }
 wasm-bindgen-futures = { version = "0.4.38", optional = true }
-web-sys = { version = "0.3.65", features = ["MessageEvent", "WebSocket"], optional = true }
+web-sys = { version = "0.3.65", features = [
+    "MessageEvent",
+    "WebSocket",
+], optional = true }
```

Comment:
Nit: Could you disable formatting on this file so that it doesn't format like this?